x/wasm/keeper/handler_plugin_encoders.go

```diff
@@ -304,13 +344,10 @@ func EncodeGovMsg(sender sdk.AccAddress, msg *wasmvmtypes.GovMsg) ([]sdk.Msg, er
 		option = govtypes.OptionNoWithVeto
 	case wasmvmtypes.Abstain:
 		option = govtypes.OptionAbstain
+	default:
+		option = govtypes.OptionEmpty
```

Comment:
Empty is not a valid vote option in the [SDK](https://github.com/cosmos/cosmos-sdk/blob/v0.45.12/x/gov/types/vote.go#L121). This will result in an error. I can make this more explicit here.